tests/phpunit/tests/plugins/test-jetpack.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-if ( version_compare( $GLOBALS['wp_version'], '5.2', '>=' ) && file_exists( DIR_TESTROOT . '/../jetpack/jetpack.php' ) ) {
+if ( version_compare( $GLOBALS['wp_version'], '5.6', '>=' ) && file_exists( DIR_TESTROOT . '/../jetpack/jetpack.php' ) ) {
```

Comment:
What happens with WordPress 5.4 and 5.5 here?

Reply:
Jetpack doesn't load.